pkg/acsengine/azureconst.go

```diff
@@ -2,7 +2,7 @@ package acsengine
 
 import "fmt"
 
-// AUTOGENERATED FILE - last generated 2017-05-19 20:39:35
+// AUTOGENERATED FILE - last generated 2017-08-04 15:59:31
```

Comment:
I agree and didn't mean to add this. I thought the other one is merged and just pushed this to my forked repo.